README.md

```diff
@@ -56,15 +56,21 @@ GitHub Enterprise repositories are not supported.
 
 ![image](/images/install02.png)
 
-3) Then click "Add tool".
+3) Depending on whether or not the repo already has a scanning tool configured, you will see "Add tool" or "Configure scanning tool". If you see "Add tool" click it and skip step 3b. If you see "Configure scanning tool" proceed to step 3b.
 
 ![image](/images/install03.png)
+![image](/images/configureScanningToolSS.png)
```

Comment:
General question about the image names — does the `SS` suffix stand for anything in particular?

In looking at the [existing content](https://github.com/ossf/scorecard-action/tree/main/images) stored under `/images/`, it looks like the general pattern is to use all-lowercase image names, so we may want to drop the `SS` and revise these new image names accordingly.